connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroDataToCatalyst.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.sql.catalyst.util.{FailFastMode, ParseMode, PermissiveMo
 import org.apache.spark.sql.errors.QueryCompilationErrors
 import org.apache.spark.sql.types._
 
-private[sql] case class AvroDataToCatalyst(
```

Comment:
4.0 added two new internal classes in this package: `AvroExpressionEvalUtils`, `AvroCompressionCodec`. Given that this package contains only internal classes and deprecated APIs since 3.0, I marked the entire `org.apache.spark.sql.avro` package as private, so that we no longer need `private[sql]` in these existing classes.